src/components/Layout/OverviewList.tsx

```diff
@@ -40,6 +43,15 @@ export default React.memo(function OverviewList(props: OverviewListProps) {
     dataPlace,
   } = props
   const l = useFormatMessage()
+
+  const isBigScreen = useMediaQuery({ minWidth: 1800 })
```

Comment:
1800 looks too arbitrary, we should probably use `1280` or `1920` which are common screen sizes

![image](https://user-images.githubusercontent.com/208789/236024507-f52e2160-3a58-4795-806d-8b74bb7bac6a.png)

also, we should also create a hook to contains that logic similar to the ones that already exists on [decentraland-ui](https://github.com/decentraland/ui/blob/master/src/components/Media/Media.ts#L6) and create an issue to move it out to this project, at least, to `decentraland-gatsby`

```suggestion
  const isBigDesktop = useBigDesktopMediaQuery()
```